sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/CentralMomentAgg.scala

```diff
@@ -194,8 +222,15 @@ case class VarianceSamp(child: Expression) extends CentralMomentAgg(child) {
 }
 
 @ExpressionDescription(
-  usage = "_FUNC_(x) - Returns the Skewness value calculated from values of a group.")
-case class Skewness(child: Expression) extends CentralMomentAgg(child) {
+  usage =
+    """
+      _FUNC_(expr) - Returns the Skewness value calculated from values of a group.
```

Comment:
Super nit but while you're changing this, there's no reason to capitalize skewness